src/main/java/mandel/projectile/Projectile.java

```diff
@@ -41,4 +41,10 @@ public double getPeakY()
     {
         return ((velocity * Math.sin(radians)) * (velocity * Math.sin(radians))) / (2 * 9.8);
     }
+
+    public double getInterceptX()
+    {
+        double time = (Math.sin(radians) * velocity) / (0.5 * 9.8);
+        return getX() + (Math.cos(radians) * velocity) * time;
```

Comment:
Why are you adding getX() to this?

Reply:
Sorry, didn't realize it was different when replying via email.
Should I use a different x value in my return statement?

Reply:
getX() doesn't return the initial position of X. It returns the current position of X at "seconds".

Reply:
In your test, getX() returns 0 since seconds is 0. 
If you change the time in your test then your test will fail.

Reply:
So if I just do **return (Math.cos(radians) * velocity) * time;** does that fix it? The test passes

Reply:
Should work

Reply:
Ok great, thank you

Reply:
I just pushed the updated code